src/SfxWeb/src/app/modules/charts/health-chart/health-chart.component.ts

```diff
@@ -114,7 +114,6 @@ export class HealthChartComponent implements OnInit, AfterViewInit, OnChanges {
     this.options.tooltip.enabled = data.length === 3;
     (this.options.series[0] as SeriesPieOptions).data = data;
 
-    this.options.title.style.fontSize =  +this.width * .2 + 'pt';
```

Comment:
whats the reason for this change?

Reply:
this is no longer a property of tile.style as of highcharts 11